options.go

```diff
@@ -142,6 +143,44 @@ func (opt *Options) init() {
 	}
 }
 
+// GetPGHost returns the host name of the database server.
+func GetPGHost() string {
+	pgHost := os.Getenv("PGHOST")
+	if pgHost == "" {
+		pgHost = "localhost"
+	}
+	return pgHost
+}
+
+// GetPGPort returns the port of the database server.
+func GetPGPort() string {
+	pgPort := os.Getenv("PGPORT")
+	if pgPort == "" {
+		pgPort = "5432"
+	}
+	return pgPort
+}
+
+// GetPGUser returns the user to be used when interacting with the database
+// server.
+func GetPGUser() string {
+	pgUser := os.Getenv("PGUSER")
+	if pgUser == "" {
+		pgUser = "postgres"
+	}
+	return pgUser
+}
+
+// GetPGDatabase returns the database name to be used when interacting with the
+// database server.
+func GetPGDatabase() string {
+	pgDatabase := os.Getenv("PGDATABASE")
+	if pgDatabase == "" {
+		pgDatabase = "postgres"
+	}
+	return pgDatabase
+}
+
```

Comment:
Can you please unexport all these functions? I don't think they should be part of the public API.

I think we don't need these functions at all - just inline the code in options init, e.g.

```
if opt.Database == "" {
    opt.Database = env("PGDATABASE", "postgres")
}

func env(key, defValue string) string {
}
```

Reply:
Thanks, done.